src/install/installDeps.ts

```diff
@@ -0,0 +1,291 @@
+      'libnspr4',
+      'libnss3',
+      'libpango-1.0-0',
+      'libpangocairo-1.0-0',
+      'libx11-6',
+      'libx11-xcb1',
+      'libxcb-dri3-0',
+      'libxcb1',
+      'libxcomposite1',
+      'libxdamage1',
+      'libxext6',
+      'libxfixes3',
+      'libxi6',
+      'libxrandr2',
+      'libxtst6'
+    ],
+    firefox: [
+      'libatk1.0-0',
+      'libcairo-gobject2',
+      'libcairo2',
+      'libdbus-1-3',
+      'libdbus-glib-1-2',
+      'libfontconfig1',
+      'libfreetype6',
+      'libgdk-pixbuf2.0-0',
+      'libglib2.0-0',
+      'libgtk-3-0',
+      'libgtk2.0-0',
+      'libpango-1.0-0',
+      'libpangocairo-1.0-0',
+      'libpangoft2-1.0-0',
+      'libx11-6',
+      'libx11-xcb1',
+      'libxcb-shm0',
+      'libxcb1',
+      'libxcomposite1',
+      'libxcursor1',
+      'libxdamage1',
+      'libxext6',
+      'libxfixes3',
+      'libxi6',
+      'libxrender1',
+      'libxt6'
+    ],
+    webkit: [
+      'gstreamer1.0-libav',
+      'gstreamer1.0-plugins-bad',
+      'gstreamer1.0-plugins-base',
+      'gstreamer1.0-plugins-good',
+      'libatk-bridge2.0-0',
+      'libatk1.0-0',
+      'libcairo2',
+      'libegl1',
+      'libenchant1c2a',
+      'libepoxy0',
+      'libfontconfig1',
+      'libfreetype6',
+      'libgdk-pixbuf2.0-0',
+      'libgl1',
+      'libgles2',
+      'libglib2.0-0',
+      'libgstreamer-gl1.0-0',
+      'libgstreamer1.0-0',
+      'libgtk-3-0',
+      'libharfbuzz-icu0',
+      'libharfbuzz0b',
+      'libhyphen0',
+      'libicu66',
+      'libjpeg-turbo8',
+      'libnotify4',
+      'libopenjp2-7',
+      'libopus0',
+      'libpango-1.0-0',
+      'libpng16-16',
+      'libsecret-1-0',
+      'libsoup2.4-1',
+      'libvpx6',
+      'libwayland-client0',
+      'libwayland-egl1',
+      'libwayland-server0',
+      'libwebp6',
+      'libwebpdemux2',
+      'libwoff1',
+      'libx11-6',
+      'libxcomposite1',
+      'libxdamage1',
+      'libxkbcommon0',
+      'libxml2',
+      'libxslt1.1'
+    ],
+  },
+};
+
+export async function installDeps() {
+  if (os.platform() !== 'linux')
+    return;
+
+  const commands: string[] = [];
+  commands.push('apt-get update');
+  const ubuntuVersion = await getUbuntuVersion();
+  let deps: any = [];
+  if (ubuntuVersion === '18.04')
+    deps = DEPENDENCIES['ubuntu18.04'];
+  else if (ubuntuVersion === '20.04')
+    deps = DEPENDENCIES['ubuntu20.04'];
+  else
+    console.warn('Cannot install dependencies for this linux distribution!');
+  console.log('Installing Ubuntu dependencies...');
+  commands.push(['apt-get', 'install', '-y', '--no-install-recommends',
+    ...deps.chromium,
+    ...deps.firefox,
+    ...deps.webkit
+  ].join(' '));
+
+  // - `ffmpeg`: For video playback in Firefox
+  commands.push('apt-get install -y ffmpeg');
+  // For headful execution
+  commands.push('apt-get install -y xvfb');
+  const child = childProcess.spawn('sudo', ['--', 'sh', '-c', `${commands.join('; ')}`], { stdio: 'inherit' });
```

Comment:
Why not childProcess.execSync() instead?

Reply:
To show progress...